tests/utils/client.py

```diff
@@ -1,6 +1,6 @@
 from steamship import Configuration, Steamship
 
-TESTING_PROFILE = "test"
+TESTING_PROFILE = "prod"
```

Comment:
is this an intentional part of the PR?

Reply:
It's not, no.

Reply:
Thanks for catching it! I do often run tests on `prod` or `staging` instead of `local`.

Reply:
this still needs to be fixed before merging, correct?